app/views/counselor/groups/show.html.erb

```diff
@@ -11,7 +11,7 @@
         <img src="<%= Faker::Avatar.image("counselor-avatar", "50x50", "png", "set2") %>">
       </div>
       <div class="col-sm-6 col-xs-6 name">
-        <%= link_to ("#{counselor.user.first_name} #{counselor.user.last_name}"), counselor_path(:id => counselor.id) %>
+        <%= link_to ("#{counselor.user.first_name} #{counselor.user.last_name}"), counselor_kid_path(:id => counselor.id) %>
```

Comment:
Is this leading to a kid's page or a counselor's page? From the looks of it, the link is put into the counselor's name.